packages/gatsby/cache-dir/__tests__/loader.js

```diff
@@ -122,36 +108,19 @@ describe(`Loader path prefixing`, () => {
 
   test(`Path prefix present and enabled`, () => {
```

Comment:
Rename this to `With custom pathPrefix`?

Reply:
I don't think so, the result is `with custom pathPrefix`, but actually we are testing with `pathPrefix` in `gatsby-config` and build with `--prefix-paths`

Reply:
👍